sunpy/map/mapbase.py

```diff
@@ -727,15 +727,16 @@ def max(self, *args, **kwargs):
     def _parse_fits_unit(unit_str):
         replacements = {'gauss': 'G',
```

Comment:
I wonder if in future we move this out somewhere and we can extend it without having to hunt for it in this file. But thats not for this PR.